tests/tasks/clean_instance.yml

```diff
@@ -0,0 +1,28 @@
+# SPDX-License-Identifier: MIT
+---
+- name: Run handlers
+  meta: flush_handlers
+
+- name: Uninstall postgresql
+  package:
+    name: "{{ __postgresql_packages }}"
+    state: absent
+
+- name: Remove data directory
+  file:
+    path: "{{ __postgresql_data_dir }}"
+    state: absent
+
+- name: Remove config files
+  file:
+    path: "{{ item }}"
+    state: absent
+  loop:
+    - "{{ __postgresql_main_conf_file }}"
```

Comment:
`__postgresql_main_conf_file` and `__postgresql_hba_conf_file` are removed twice. Both of them are located in previously deleted `__postgresql_data_dir` but it shouldn't be any issue

Reply:
Right - it just reports that the files were already removed (`"changed": false`)
```
TASK [Remove config files] *****************************************************
task path: /home/rmeggins/linux-system-roles/postgresql/tests/tasks/clean_instance.yml:17
Tuesday 02 May 2023  17:02:51 -0600 (0:00:00.506)       0:01:21.590 *********** 
ok: [/home/rmeggins/.cache/linux-system-roles/rhel-8-9.qcow2] => (item=/var/lib/pgsql/data/postgresql.conf) => {
    "ansible_loop_var": "item",
    "changed": false,
    "item": "/var/lib/pgsql/data/postgresql.conf",
    "path": "/var/lib/pgsql/data/postgresql.conf",
    "state": "absent"
}
ok: [/home/rmeggins/.cache/linux-system-roles/rhel-8-9.qcow2] => (item=/var/lib/pgsql/data/pg_hba.conf) => {
    "ansible_loop_var": "item",
    "changed": false,
    "item": "/var/lib/pgsql/data/pg_hba.conf",
    "path": "/var/lib/pgsql/data/pg_hba.conf",
    "state": "absent"
}
changed: [/home/rmeggins/.cache/linux-system-roles/rhel-8-9.qcow2] => (item=/etc/postgresql/system-roles.conf) => {
    "ansible_loop_var": "item",
    "changed": true,
    "item": "/etc/postgresql/system-roles.conf",
    "path": "/etc/postgresql/system-roles.conf",
    "state": "absent"
}
changed: [/home/rmeggins/.cache/linux-system-roles/rhel-8-9.qcow2] => (item=/etc/postgresql/system-roles-internal.conf) => {
    "ansible_loop_var": "item",
    "changed": true,
    "item": "/etc/postgresql/system-roles-internal.conf",
    "path": "/etc/postgresql/system-roles-internal.conf",
    "state": "absent"
}
```
doing it this way, if the location of `__postgresql_main_conf_file` or `__postgresql_hba_conf_file` is changed to something other than `__postgresql_data_dir`, we will not have to update the code here.